build_deps/build_pip_pkg.sh

```diff
@@ -77,7 +82,11 @@ function main() {
 
   BUILD_CMD="setup.py bdist_wheel --platlib-patch"
   if is_macos; then
-    BUILD_CMD="${BUILD_CMD} --plat-name macosx_10_13_x86_64"
+    if is_arm64; then
+      BUILD_CMD="${BUILD_CMD} --plat-name macosx_12_0_arm64"
+    else
+      BUILD_CMD="${BUILD_CMD} --plat-name macosx_10_13_x86_64"
```

Comment:
Also update to macosx_12_0?

Reply:
It is stated that "Requirements: MacOS Monterey 12.0.0"

Reply:
Yes, it's a requirement for macOS arm64 platform, but not for x86's macOS.